api/drivers/qemu/driver.js

```diff
@@ -41,13 +42,22 @@ class QemuDriver extends Driver {
   initialize() {
     return ConfigService.get('qemuServiceURL', 'qemuServicePort')
       .then((config) => {
-
-        let requestOptions = {
-          url: 'http://' + config.qemuServiceURL + ':' + config.qemuServicePort,
-          method: 'GET'
-        };
-
-        return request(requestOptions);
+        Image.update({
+          default: true
+        }, {
+          iaasId: 'Qemu',
+          name: 'image.qcow2',
```

Comment:
You have to switch *iaasId* and *name* usage.
Default image name should be something like "Qemu default image"  and iaasId should match the path of this default image in qemumanager container